### ISSUE TITLE
Magento widgets bug Tabs with Accordion

### ISSUE THREAD

**Issue by Naberdz** [magento/magento2#22819]
### Preconditions (*)

1. Magento 2.3.1
2. Magento 2.3-develop

### Steps to reproduce (*)

1. Create tabs widget as described here:  https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_tabs.html
`data-mage-init='{"mage/tabs": {"openedState": "active", "animate": {"duration": 100}, "active": 0}}'`
2. Create accordion widget https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_accordion.html and place it in tab widget
`data-mage-init='{"accordion":{"collapsible": true,"multipleCollapsible": true}}'`

### Expected result (*)

1. Both widgets must work as intended tabs opened by click on tab title and accordion  by clicking on accordion title

### Actual result (*)

1. If we click accordion items it closes tab.
```
<div class="tabs" data-role="tablist">
	<div class="tab title" data-role="collapsible" role="presentation" data-collapsible="true">
		<a class="switch" data-toggle="trigger" href="#tab-simple">Tab Simple</a>
	</div>
	<div class="tab title" data-role="collapsible" role="presentation" data-collapsible="true">
		<a class="switch" data-toggle="trigger" href="#tab-advanced">Tab Advanced</a>
	</div>
	<div id="tab-simple" data-role="content" role="tabpanel" aria-hidden="true">
		tab content
	</div>
	<div id="tab-advanced" data-role="content" role="tabpanel" aria-hidden="true">
		<div id="accordion" role="tablist">
			<div data-role="collapsible" role="presentation" data-collapsible="true" aria-disabled="false">
				<div data-role="title" role="tab" aria-selected="false" aria-expanded="false" tabindex="0" aria-controls="tab-simple">
					<h4>Title 1</h4>
				</div>
				<div data-role="content" role="tabpanel" aria-hidden="true" style="display: none;">
					<div class="form-group">
						Some content
					</div>    				
				</div>
			</div>
			<div data-role="collapsible" role="presentation" data-collapsible="true" aria-disabled="false">
				<div data-role="title" role="tab" aria-selected="false" aria-expanded="false" tabindex="0" aria-controls="tab-advanced">
					<h4>Title 2</h4>
				</div>
				<div data-role="content" role="tabpanel" aria-hidden="true" style="display: none;">
					<div class="form-group">
						Some content
					</div> 
				</div>
			</div>
		</div>
	</div>
</div>
```

**Comment by engcom-Delta** [comment]
Hi @Naberdz thank you for your report. Unfortunately,  I am not able to reproduce issue by steps you described on clean 2.3-develop
![22819issue](https://user-images.githubusercontent.com/51681379/67935424-71a73c00-fbd2-11e9-90d8-a88f65205428.gif)

If you'd like to update the issue, please reopen it.

**Comment by Naberdz** [comment]
@engcom-Delta 
You have to place accordion in tabs..  right now you have tabs and accordion as standalone items.

**Comment by engcom-Delta** [comment]
@Naberdz thanks for update.
Issue is reproducible on 2.3-develop if add `Accordion widget` to `Tabs widget` 
![#20819issue_2](https://user-images.githubusercontent.com/51681379/67946512-344ea880-fbea-11e9-919a-70e3d35d1dd7.gif)

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Delta 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-22619` were created

**Issue Available**: @engcom-Delta, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by ptylek** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [Magento instance](https://i-22819-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22819-2-3-develop.instances.magento-community.engineering/admin](https://i-22819-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @Naberdz. Thank you for your report. 
The issue has been fixed in magento/magento2#25515 by @ptylek in 2.3-develop branch
Related commit(s):
 - [8132944e9942a0d46811d99127f1706cdd8109ed](https://github.com/magento/magento2/commit/8132944e9942a0d46811d99127f1706cdd8109ed)
 - [322709d1572caec44189edb1e8b3ca5f0be628fc](https://github.com/magento/magento2/commit/322709d1572caec44189edb1e8b3ca5f0be628fc)
 - [d13cd157c4abc215fea2c23f5be63e10ccead415](https://github.com/magento/magento2/commit/d13cd157c4abc215fea2c23f5be63e10ccead415)
 - [b4ec7287182fdea8ef89c90e3cf991f7888fff43](https://github.com/magento/magento2/commit/b4ec7287182fdea8ef89c90e3cf991f7888fff43)
 - [19432787ab80d80b0dc23b10e26bbc64b67b9da2](https://github.com/magento/magento2/commit/19432787ab80d80b0dc23b10e26bbc64b67b9da2)

The fix will be available with the upcoming 2.3.5 release.
